sbids/metrics/c2st.py

```diff
@@ -0,0 +1,128 @@
+
+from tqdm import tqdm
+import numpy as np
+import jax
+import jax.numpy as jnp
+import haiku as hk
+import optax
+from tensorflow_probability.substrates import jax as tfp
+tfd = tfp.distributions
+tfb = tfp.bijectors
+
+class Classifier_C2ST(hk.Module):
```

Comment:
Is this following the architecture of the SBIBM paper? If so, we can add one line of comment here to say what we based our model on.

Reply:
nop it's not the same, their classifier: 
```python 
    clf = MLPClassifier(
        activation="relu",
        hidden_layer_sizes=(10 * ndim, 10 * ndim),
        max_iter=10000,
        solver="adam",
        random_state=seed,
    )
```

Reply:
So, why is it different ^^?

Reply:
seems that I have good results now, I change it